### PR TITLE
Feat/new settings

### PR DESCRIPTION
**Pull request by shadow2kk**

### CODE REVIEW

**Review by LucasM19 — APPROVED** [review]

File: apps/app/components/Settings/SettingAddress.tsx [hunk 1]
```diff
@@ -44,26 +49,77 @@ import {
...
+  Default = 0,
+  Ens = 1,
+  Bit = 2,
+  More = 3,
+}
+
+const tabsConfig: Record<
+  TabItemType,
+  {
+    Icon: any
```
Comment (Codemasker) [inline apps/app/components/Settings/SettingAddress.tsx:85]: The initial should be lowercase, Type should be `ReactNode`

File: apps/app/components/Settings/SettingAddress.tsx [hunk 2]
```diff
@@ -323,252 +389,410 @@ export const SettingAddress: React.FC = () => {
...
-      </FormControl>
-      <Flex className="mascot">
-        <Mascot src={happySetupMascot} />
-      </Flex>
+          </Center>
+
+          <FormControl>
+            <Flex justifyContent="center" pt="8px" minH="200px">
+              <TabPanels maxW="480px">
+                {[
```
Comment (Codemasker) [inline apps/app/components/Settings/SettingAddress.tsx:574]: This `TabItemType` can be written as a variable.
Like:
```tsx
const isEnabledMore = false
const tabItemTypes = [
  TabItemType.Default,
  TabItemType.Ens,
  TabItemType.Bit,
  ...(isEnabledMore ? [TabItemType.More] : [])
]

tabItemTypes.map(type => 
  // make jsx
)
```

File: apps/app/components/Settings/SettingAddress.tsx [hunk 3]
```diff
@@ -323,252 +389,410 @@ export const SettingAddress: React.FC = () => {
...
+                  TabItemType.Default,
+                  TabItemType.Ens,
+                  TabItemType.Bit,
+                  // TabItemType.More,
+                ].map((type) => {
```
Comment (Codemasker) [inline apps/app/components/Settings/SettingAddress.tsx:579]: This map function is too long. You can separate JSX into components.

File: apps/app/i18n/en.json [hunk 4]
```diff
@@ -220,7 +226,11 @@
             "register-bit": "Register <a>new</a> .bit",
             "request-failed": "Request change default address failed, please try again later.",
             "refresh": "Refresh",
-            "refresh_failed": "Refresh Failed: "
+            "refresh_failed": "Refresh Failed: ",
+            "not-found-p1": "Opos!!<br><br/>Cannot associate to your domain name",
```
Comment (Codemasker) [inline apps/app/i18n/en.json:230]:
```suggestion
            "not-found-p1": "Opos!!<br/>Cannot associate to your domain name",
```

**Review by ghcoldsnap — APPROVED** [review]

File: apps/app/i18n/en.json (hunk 4 above)
Comment (LucasM19) [reply apps/app/i18n/en.json:230]: also there is a typo? Opos -> Oops

**Pull request merged**